onboarding/views.py

```diff
@@ -336,11 +335,31 @@ class ContactFormViewSet(viewsets.ModelViewSet):
     queryset = ContactRequestDetail.objects.all()
     serializer_class = ContactFormTestSerializer
 
-class AddUserToPackageViewSet(viewsets.ModelViewSet):
 
-    queryset = Package.objects.all()
-    serializer_class = PackageAddUsersSerializer
-    permission_classes = [IsAuthenticated]
+class PackagesUsersViewSet(viewsets.ModelViewSet):
```

Comment:
Logic here is fine, we get to the package by "hub" model PackageUser.

Reply:
So called `through model` :wink: 
https://docs.djangoproject.com/en/3.1/topics/db/models/#extra-fields-on-many-to-many-relationships